apps/myjobs/Gemfile

```diff
@@ -2,11 +2,11 @@ source 'https://rubygems.org'
 
 
 # Bundle edge Rails instead: gem 'rails', github: 'rails/rails'
-gem 'rails', '5.2.7.1'
+gem 'rails', '6.0.5'
 # Use sqlite3 as the database for Active Record
 # Note that the version has been fixed due to an incompatibilty
 # https://github.com/rails/rails/issues/35153
-gem 'sqlite3', '1.3.13'
+gem 'sqlite3', '1.4.2'
```

Comment:
I'll have to check to re-confirm, but this got updated because rails and/or dependencies needed it. I'll look into it further.

Reply:
Ubuntu 22.04 *

Reply:
It was only happening on Ubuntu 22.04 during build of OnDemand package that are part of ondemand-packaging E2E tests.  Google turned up issue with people reporting exact same symbol error during gem build and newer gem solved the issue.

Reply:
No I meant in the past on RHEL I recall having to load the sqlite3 module.  But just from my memory - it could have actually been i needed the module to open a sqlite3 file? I don't recall correctly.

Why were we shipping our own sqlite3 lib?

Reply:
Oh, we needed newer sqlite3 for RHEL7.  For RHEL8 and Ubuntu we don't ship sqlite3 library.

Reply:
You're right @Oglopf - I had the module loaded to build, but even so - it's the runtime version that matters, that's why it works within the PUN because that sqlite3.so in the second ldd there is >= 3.8.

All you have to do to replicate is purge that module and try a `bin/bundle exec rake db:prepare` and you get this.
```
Your version of SQLite (3.7.17) is too old. Active Record supports SQLite >= 3.8.
/users/PZS0714/johrstrom/ondemand/src/apps/myjobs/vendor/bundle/ruby/2.7.0/gems/activerecord-6.0.5/lib/active_record/connection_adapters/sqlite3_adapter.rb:341:in `check_version'
/users/PZS0714/johrstrom/ondemand/src/apps/myjobs/vendor/bundle/ruby/2.7.0/gems/activerecord-6.0.5/lib/active_record/connection_adapters/abstract/connection_pool.rb:888:in `block in new_connection'
```

Reply:
> It accepts the system installed libsqlite3 as a compilation target. And at runtime - it's a higher version.

That's why this works.